inception-scheduling/src/main/java/de/tudarmstadt/ukp/inception/scheduling/SchedulingService.java

```diff
@@ -36,19 +37,18 @@
 {
     private static final Logger log = LoggerFactory.getLogger(SchedulingService.class);
 
-    private static final int NUMBER_OF_THREADS = 4;
-    private static final int QUEUE_SIZE = 100;
-
     private final ApplicationContext applicationContext;
     private final ThreadPoolExecutor executor;
 
     private final List<Task> runningTasks;
 
     @Autowired
-    public SchedulingService(ApplicationContext aApplicationContext)
+    public SchedulingService(ApplicationContext aApplicationContext,
+                             @Value("${scheduler.threads:4}") int numberOfThreads,
```

Comment:
Hm... I'm always using a dedicated class to contain the configuration (e.g. `KnowledgeBaseProperties`)... I tend to find that more "scalable" when adding more settings in the future. Due to #780 I am also now switching all of these classes to an interface/impl pair.